sympy/solvers/ode.py

```diff
@@ -7540,7 +7542,7 @@ def _linear_3eq_order1_type3(x, y, z, t, r):
     .. math:: a^2 C_1 + b^2 C_2 + c^2 C_3 = 0
 
     """
-    C0, C1, C2, C3 = symbols('C0:4')
+    C0, C1, C2, C3 = get_numbered_constants(Eq(x(t)), num=4, start=0)
```

Comment:
why does 3 equations have 4 parameters anyway?   (I haven't looked at it it, just warning you not to trust these codes as correct)